sdk/batch/Azure.ResourceManager.Batch/api/Azure.ResourceManager.Batch.netstandard2.0.cs

```diff
@@ -1202,4 +1208,43 @@ public enum NodeCommunicationMode
         Classic = 1,
         Simplified = 2,
     }
+    public partial class OSDisk
```

Comment:
this name is colliding with the type in compute library, could we rename this to prepend a `Batch` prefix?